template/package.json

```diff
@@ -3,25 +3,27 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
+    "android": "react-native run-android",
+    "ios": "react-native run-ios",
     "start": "react-native start",
     "test": "jest"
   },
   "dependencies": {
-    "react": "16.8.6",
-    "react-native": "0.60.5"
+    "react": "16.8.1",
+    "react-native": "0.61.1"
   },
   "devDependencies": {
     "@babel/core": "^7.5.0",
     "@babel/runtime": "^7.5.0",
     "@react-native-community/eslint-config": "^0.0.3",
     "@types/jest": "^24.0.18",
-    "@types/react": "^16.9.2",
-    "@types/react-native": "^0.60.14",
+    "@types/react": "^16.9.3",
```

Comment:
Maybe we should align @types version with React version? We might end up in a state when types are incompatible with the version of React that we use

Reply:
I thought so as well, I just wasn't sure if the `@types` versions always match the library versions. But I think now I understood the version naming pattern 😄